docs/infrastructure/deployment-targets/cloud-target-discovery/index.md

```diff
@@ -87,8 +99,43 @@ To discover targets for an environment, dynamic infrastructure needs to be enabl
 Using target discovery during a deployment means that there may be no existing targets at the start of a deployment. To allow deployments to start without any targets:
 
 1. Navigate to **{{Projects,Project name,Deployments,Settings}}**.
-1. Expand the **Deployment Targets Required** and select the **Allow deployments to be created when there are no deployment targets** option.
-1. Click **SAVE**.
+2. Expand the **Deployment Targets Required** and select the **Allow deployments to be created when there are no deployment targets** option.
+3. Click **SAVE**.
+
+## Discovering existing targets
+
+Cloud Target Discovery will often discover resources which already have targets in Octopus. Here are some things to be aware of: 
+
+### Previously discovered targets
+
+If a target has been created via Cloud Target Discovery, the next time the same cloud resource is discovered, the target will simply be updated. Targets are matched by to the resource by target name, which is formatted depending on the discovered resource. The names are chosen to be unique but as readable as possible.
+
+- Azure Web App: `azure-web-app/{resource-group}/{web-app-name}`
+- ECS Cluster: `{ecs-cluster-arn}`
+- AKS Cluster: `aks/{subscription-id}/{resource-group}/{cluster-name}`
+- EKS Cluster: `{eks-cluster-arn}`
+
+:::warning
+Renaming or moving cloud resources can cause target discovery to create duplicate targets. In most cases the old target will become unhealthy and be removed automatically by Octopus (see [Cleaning up unhealthy targets]) but in some cases the old target may still be healthy. In these cases, it must be removed manually.
+
+**Example:** If you move an AKS Cluster to a different subscription and then update your Account in Octopus to use the new subscription ID, the old target will still pass its health-check. When discovery occurs a new target will be created (with the new Subscription ID in the target name) and the old target will need to be removed manually.
+:::
+
+### Overwriting manually added targets
+
+Manually added targets can be overridden by Cloud Target Discovery but this will only happen if they match the name format above exactly. If they are different, a new target will be created.
+
+:::hint
+We recommend avoiding manually added targets when using Cloud Target Discovery to avoid duplicate targets for a single cloud resource.
```

Comment:
```suggestion
We recommend not manually adding targets when using Cloud Target Discovery to avoid duplicate targets for a single cloud resource.
```